django/core/checks/security/base.py

```diff
@@ -1,16 +1,30 @@
+import itertools
+
 from django.conf import settings
 
-from .. import Tags, Warning, register
+from .. import Error, Tags, Warning, register
+
+FEATURE_POLICY_DIRECTIVES = {
+    'accelerometer', 'ambient-light-sensor', 'autoplay', 'camera', 'display-capture', 'document-domain',
+    'encrypted-media', 'fullscreen', 'geolocation', 'gyroscope', 'layout-animations', 'legacy-image-formats',
+    'magnetometer', 'microphone', 'midi', 'oversized-images', 'payment', 'picture-in-picture', 'speaker', 'sync-xhr',
+    'unoptimized-images', 'unsized-media', 'usb', 'vibrate', 'vr', 'wake-lock', 'webauthn',
+}
+
+REFERRER_POLICY_VALUES = {
+    'no-referrer', 'no-referrer-when-downgrade', 'origin', 'origin-when-cross-origin', 'same-origin', 'strict-origin',
+    'strict-origin-when-cross-origin', 'unsafe-url',
```

Comment:
Please wrap at 79 chars.

Reply:
(Just because I keep seeming to get this wrong... I thought that we allowed for 119 characters except for docstrings, comments and multi-line import statements. Has that changed or ...?)